SysTableParse.py

```diff
@@ -123,144 +154,194 @@ def cardtotex(card):
 			r_b=float(relative_b)
 			r_s=float(relative_s)
 			sysname = vals[0]
-			sysname = sysname.replace('NORM',' Normalization')
-			if 'SHAPE' in sysname:
-				sysname = sysname.replace('SHAPE','') + ' Shape'
+			#sysname = sysname.replace('NORM',' Normalization')
+			#if 'SHAPE' in sysname:
+			#	sysname = sysname.replace('SHAPE','') + ' Shape'
 			sysnames.append(sysname)
 			#if float(mass)>1500 : continue
 			if gotToBetaHalf==True : continue #== for mumujj, != for munujj, commented out for both
 			#print sysname
                         #if mass=='2300': continue
                         #if mass=='2900': continue
-			if 'TT Shape' in sysname:
+			if sysname == 'LUMICorr': 
```

Comment:
I think we should only report back one LUMI systematic, even if they are processed separately. You can just add them in quadrature sqrt(a**2 + b**2....). You should do that before you compare the min/max. Then in the table only one line for LUMI

Reply:
So, we discussed maybe doing this for the "range" table, but for the full systematic tables we would just have all the systematics from the datacards and put an xx where not applicable (since we combine the three-years into one table).